clusterloader2/api/types.go

```diff
@@ -41,8 +41,8 @@ type TestScenario struct {
 type Config struct {
 	// Name of the test case.
 	Name string `json: name`
-	// AutomanagedNamespaces is a number of automanaged namespaces.
-	AutomanagedNamespaces int32 `json: automanagedNamespaces`
+	// Namespace is a structure for namespace configuration.
+	Namespace *NamespaceConfig `json: namespace`
```

Comment:
We can't do that in a single step. We should:
- add this field and mark the AutomanagedNamespaces field as deprecated (though still respect it somehow)
- allow people for doing the transition
- only in some time get rid of this field

Reply:
sure,  we'll keep it as as deprecated.

Reply:
The following default values are used if a parameter is not specified in either new or old way. 
```
     count: 1
     prefix: ""
     deleteStaleNamespaces: false
     deleteAutomananagedNamespaces: true
```

Reply:
AutomanangedNamespaces is kept.  If the number of namespaces is specified by both AutoMananagedNamespaces and the new structure, the value of new structure will be used. If neither defines the parameter, a default value of 1 will be used.  The new code also checks the value and reports an error if it's less than 1.